### ISSUE TITLE
Mode discret

### ISSUE THREAD

**Issue by Dites33** [redbug26/jeedom-tahoma#22]
Bonjour

Avez vous la possibilité de travailler sur l'ajout du mode discret ? 
Le mode discret a plusieurs avantage pour moi. Il me permet d'actionner les volets pendant le sommeil de mes enfants sans les réveiller et surtout, il me permet d'actionner mes volets des pièces de vie sans déclencher mes alarmes (qui réagissent si les volets descendent trop vite).

Cela serait un grand plus et il me semble avoir vu dans les tickets que cette fonction était appelable via les API de Somfy

En vous remerçiant par avance.

**Comment by nicolas-albert** [comment]
Bonjour,
J'ai vu le paramètre qui manque dans la requête. Si je trouve le temps, je ferais un fork pour l'ajouter.
Le truc qui me dérange le plus, c'est l'interface.
Il faudrait presque doubler toutes les commandes, pour être "discret" (up discret, down discret, slider discret ...).

**Comment by Dites33** [comment]
Bonjour @nicolas-albert, si les commandes existent mais ne sont pas en 'afficher' par défaut, cela nous permettrait de choisir si nous souhaitons charger notre interface ou non  (ou de remplacer les commandes 'normales', par les commandes discretes surtout que tout les volets somfy n'ont pas ce mode, j'ai un grand volet qui ne l'a pas alors que mes 9 autres l'ont bien.

**Comment by nicolas-albert** [comment]
Oui, ça permettrait de faire du cas par cas. Je penserai à en parler ici si je me lance un jour.

**Comment by Dites33** [comment]
Merci, ca serait top, c'est le seul truc qui manque pour une utilisation à 100 % de mon côté.
J'en profite pour te remercier pour ton travail !

**Comment by nicolas-albert** [comment]
J'ai rien fait pour l'instant ! Je ne suis qu'un simple utilisateur qui découvre.
Là par exemple, je viens de réussir à piloter un volet via Google Assistant en passant par IFTTT et Jeedom, mais je crois que jeedom-tahoma ne met pas toujours bien à jour ses cookies d'authentification.

**Comment by Gloumi** [comment]
Hello ! Je remonte ce sujet, je suis également intéressé par cette fonctionnalité, très pratique notamment pour des scénarios de réveil. 
Je veux bien donner un coup de main pour le développement de la fonctionnalité, mais il faudra me dire où ça se passe, je n'ai pas l'impression que le code soit sur ce repo.

**Comment by jptstar** [comment]
Salut a tous,
Tous d'abord merci à Redbug pour ce Plugin!!

J'avais le même soucis que vous tous concernant les volets RS100.
J'ai modifié le code du plugin et chez moi ca marche super.

Le code est certainement optimisable, mais fonctionnel.
Alors si Redbug est d'accord, il pourrait l'intègrerai au plugin et je peux lui envoyer.

Par contre, j'ai du supprimer les volets RS100 du plugin Tahoma et les resynchroniser pour afficher les nouvelles commandes.

<img width="603" alt="Capture d’écran 2020-07-31 à 12 47 59" src="https://user-images.githubusercontent.com/25709145/89028186-228e6300-d32c-11ea-8eab-9edf4ae5bcab.png">

**Comment by redbug26** [comment]
Bonjour @jptstar, je suis effectivement interressé par ta modif ;)

**Comment by jptstar** [comment]
Bonjour redbug,

Voici le code (ci-joint) que j’ai modifié pour la fonction plume des volets io Somfy.
Le code se trouve entre des commentaires ////NEW CODE ///
Ce n’est pas optimal mais ça marche bien.

Merci de bien vouloir l’intégrer à ton code.

Merci et a bientôt.

Jptstar
